folio-spring-system-user/src/main/java/org/folio/spring/service/SystemUserProperties.java

```diff
@@ -1,7 +1,13 @@
 package org.folio.spring.service;
 
+import org.apache.commons.lang3.StringUtils;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 @ConfigurationProperties("folio.system-user")
 public record SystemUserProperties(String username, String password, String lastname, String permissionsFilePath) {
+  public SystemUserProperties {
+    if (StringUtils.isEmpty(password)) {
```

Comment:
Considering what this PR is about, wouldn't `isBlank` be more appropriate here? I can imagine a mistake or someone deliberately overriding this by providing a string only with whitespace.

Reply:
Ok an all whitespace password would be a very strange password but I get your point. I think there are other considerations here though, which concern how we validate system user passwords in general, which probably do require a separate ticket and separate thinking.

I do think that allowing any whitespace in a password is generally not a good idea, considering how easy it is to accidentally include a whitespace character, but then not be able to see it, and struggle to understand why the password doesn't work.

I'm also concerned wondering how we validate these system user passwords in general, if at all. I know we have mod-password-validator which performs this purpose for mod-users-bl. But is it (or should it) be possible to have system users with passwords like "password" or "system"? Without validation you could obviously end up with this unfortunate situation.